drivers/epd_bw_spi/include/epd_bw_spi_params.h

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2019 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_epd_bw_spi
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for epd_bw_spi
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ */
+
+#ifndef EPD_BW_SPI_PARAMS_H
+#define EPD_BW_SPI_PARAMS_H
+
+#include "board.h"
+#include "epd_bw_spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief    Set default configuration parameters for a SPI e-Paper display
+ * @{
+ */
+#ifndef EPD_BW_SPI_PARAM_SPI
+#define EPD_BW_SPI_PARAM_SPI        (SPI_DEV(0))
+#endif
+#ifndef EPD_BW_SPI_PARAM_SPI_CLK
+#define EPD_BW_SPI_PARAM_SPI_CLK    (SPI_CLK_5MHZ)
+#endif
+#ifndef EPD_BW_SPI_PARAM_CS
+#define EPD_BW_SPI_PARAM_CS         (GPIO_UNDEF)
```

Comment:
```suggestion
#define EPD_BW_SPI_PARAM_CS         (SPI_CS_UNDEF)
```

I think `SPI_CS_UNDEF` could be used here